students/21th/login.js

```diff
@@ -0,0 +1,31 @@
+const inputId = document.getElementById('id');
+const inputPw = document.getElementById('password');
+const inputItems = document.getElementById('inputContainer')
+const logInButton = document.getElementById('logInButton');
+const mainATag = document.getElementById(moveMain);
+const loginEventTrue = () => {
+    logInButton.disabled = false;
+    logInButton.style.cursor="pointer";
+    logInButton.style.backgroundColor = "#0095f6";
+    mainATag.disabled = false;
+};
+const loginEventFalse = () => {
+    logInButton.disabled=true;
+    logInButton.style.cursor="default";
+    logInButton.style.backgroundColor = "#B9DFFC";
+}
+
+
+inputItems.addEventListener('keyup', (event)=>{
+    inputId.value.length >= 1 && inputPw.value.length >= 1 ?
+    loginEventTrue():loginEventFalse();
+});
```

Comment:
```suggestion
inputItems.addEventListener('keyup', (event)=>{
    const isInputValid = inputId.value.length >= 1 && inputPw.value.length >= 1
    
    isInputValid ? loginEventTrue() : loginEventFalse();
});
```
조건에 변수명을 지정해주시면 보다 가독성 있는 로직을 작성할 수 있습니다!

Reply:
감사합니다! 앞으로 작성할 부분에도 적용하겠습니다